packages/containers/Import/Steps/DataStep/utils.js

```diff
@@ -0,0 +1,22 @@
+import memoize from 'memoize-one';
+import { SEARCH_FIELDS } from './constants';
+
+export const capitalize = string => {
+  if (typeof string !== 'string') return '';
```

Comment:
`capitalize(33) => ''`
`capitalize('33') => '33'`

Perhaps, it's not valid for your case) 
// nitpick

Reply:
Why not? If a string contains just numbers, so we do not need to capitalize it?